src/analysis/retail/evoker/preservation/modules/talents/DreamBreath.tsx

```diff
@@ -107,12 +112,21 @@ class DreamBreath extends Analyzer {
     const entries: BoxRowEntry[] = [];
     this.casts.forEach((cast) => {
       let value = QualitativePerformance.Fail;
+      const groupSize = Object.keys(this.combatants.getEntities()).length;
       const coyActive =
         this.selectedCombatant.hasTalent(TALENTS_EVOKER.CALL_OF_YSERA_TALENT) && cast.coyActive;
-      if (coyActive && cast.targetsHit === 5) {
-        value = QualitativePerformance.Good;
-      } else if (coyActive && cast.targetsHit === 4) {
-        value = QualitativePerformance.Ok;
+      if (groupSize > 5) {
```

Comment:
Agree with Hana that you need to check COY. It also could be a lot cleaner if you just set a variable called

`
const maxTargets = groupSize > 5 ? 6 : 5;
`

and then you can just say `if (cast.targetsHit == maxTargets) ... else if (cast.targetsHit == maxTargets - 1)...`